charts/ccd-case-management-web/values.yaml

```diff
@@ -1,17 +1,14 @@
 nodejs:
-  image: hmctspublic.azurecr.io/ccd/case-management-web:latest
-  applicationPort: 3451
-  memoryRequests: 256Mi
```

Comment:
how come memoryRequests, cpuRequests, memoryLimits have been removed?

Reply:
lot of things are moved into base images like java or nodes, hence no need in sub charts

Reply:
👍